nc-news/src/components/AllArticles.jsx

```diff
@@ -0,0 +1,33 @@
+import { Link } from "react-router-dom";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faComment, faThumbsUp } from "@fortawesome/free-solid-svg-icons";
+import { Box } from "./styled-components/Box";
+
+export default function AllArticles({ allArticles }) {
+  return (
+    <section className="articles">
+      <h2>Articles</h2>
+      {allArticles.map((article) => {
+        return (
+          <Box>
+            <Link to="/SingleArticle">
+              <h3>{article.title}</h3>
+            </Link>
+            <p>Posted by {article.author}</p>
+            <img src={article.article_img_url} />
+            <div className="articles__box-footer">
+              <div className="articles__box-footer--comments">
+                <FontAwesomeIcon icon={faComment} size="lg" />{" "}
+                {article.comment_count}
+              </div>
+              <div className="articles__box-footer--votes">
+                <FontAwesomeIcon icon={faThumbsUp} /> {article.votes}
+              </div>
+            </div>
+          </Box>
```

Comment:
this could be seperated into a ArticleCard component in its own file to neaten up and de-bloat the AllArticles.jsx file.